doc/manual/manual.tex

```diff
@@ -5365,6 +5365,91 @@ \subsubsection{Tracking finite strain}
 they can easily be used for computing material model outputs such as the viscosity.  
 
 
+\subsubsection{Reading in compositional initial composition files generated with geomIO}
+\label{sec:geomio}
+\textit{This section was contributed by Juliane Dannberg}
+
+Many geophysical setups require initial conditions with several different materials and complex geometries. 
+Hence, sometimes it would be easier to generate the initial geometries of the materials as a drawing instead 
+of by writing code. The MATLAB-based library geomIO (\url{http://geomio.bitbucket.org/}) provides a convenient tool
+to convert a drawing generated with the vector graphics editor Inkscape (\url{https://inkscape.org/en/}) to a data 
+file that can be read into \aspect{}. Here, we will demonstrate how this can be done for a 2d setup for a model
+with one compositional field, but geomIO also has the capability to create 3D volumes based on series of 2D vector 
+drawings using any number of different materials; and the initial conditions could also be used with particles 
+instead of compositional fields. 
+
+To obtain the developer version of geomIO, you can clone the bitbucket repository by executing the command
+\begin{verbatim}
+ git clone https://bitbucket.org/geomio/geomio.git
+\end{verbatim}
+or you can download geomIO \href{https://bitbucket.org/geomio/geomio/downloads}{here}. 
+You will then need to add the geomIO source folders to your MATLAB path by running the file located in
+\texttt{/path/to/geomio/installation/InstallGeomIO.m}.
+An extensive documentation for how to use geomIO can be found \href{http://geomio-doc.bitbucket.org/}{here}.
+Among other things, it explains \href{http://geomio-doc.bitbucket.org/tuto2D.html#drawing}{how to generate drawings in Inkscape}
+that can be read in by geomIO, which involves assigning new attributes to paths in Inkscape's XML editor. 
+In particular, a new property `phase' has to be added to each path, and set to a value corresponding to the 
+index of the material that should be present in this region in the initial condition of the geodynamic model.   
+
+\note{geomIO currently only supports the latest stable version of Inkscape (0.91), and other versions might not
+  work with geomIO or cause errors. Moreover, geomIO currently does not support grouping paths (paths can still 
+  be combined using \texttt{Path$\rightarrow$Union}, \texttt{Path$\rightarrow$Difference} or similar commands), 
+  and only the outermost closed contour of a path will be considered. This means that i.e. for modelling a spherical 
```

Comment:
did you mean "e.g." instead of "i.e."? If so, write it as "This means that, for example, for modeling..." (modeling also with one 'l' in American English).